src/Network/Network/Models/PSApplicationGatewayFirewallPolicySettings.cs

```diff
@@ -42,5 +42,8 @@ public partial class PSApplicationGatewayFirewallPolicySettings
 
         [Ps1Xml(Target = ViewControl.Table)]
         public int? CustomBlockResponseStatusCode { get; set; }
+
+        [Ps1Xml(Target = ViewControl.Table)]
+        public PSApplicationGatewayFirewallPolicySettingLogScrubbingConfiguration LogScrubbing { get; set; }
```

Comment:
For the sake of consistency, could you please follow the same naming convention as is on NRP? 
[Link to NRP](https://msazure.visualstudio.com/One/_git/Networking-nrp?path=/src/sources/Frontend/FrontendContract/Csm.Public/ApplicationGatewayWebApplicationFirewallPolicySettings.cs&version=GC7ea27728df562cad1bb38502dbc7a300c5517834&line=52&lineEnd=53&lineStartColumn=1&lineEndColumn=1&lineStyle=plain&_a=contents)

Reply:
I think this is the correct naming convention for powershell, like the other files for applicationgateway firewall, but I will check

Reply:
Naming convention for log scrubbing config

Reply:
It is called log scrubbing config in NRP I believe

Reply:
changed name according to Tejas's comment, this need to be identical to swagger, not nrp though